ERC721/serverless-NFT/1-01.md

```diff
@@ -92,6 +83,8 @@ Compiling ./contracts/Sample.sol...
   1 passing (171ms)
 ```
 
+<!-- TODO: Is this needed? -->
```

Comment:
it's needed to auto-deploy to netlify in part 2 but could also only occur there when it's necessary